vhdbuilder/packer/components.json

```diff
@@ -12,7 +12,8 @@
       "amd64OnlyVersions": [],
       "multiArchVersions": [
         "v0.5.2",
-        "v0.6.1"
+        "v0.6.1",
```

Comment:
Can these older versions be removed? They have CVEs and were flagged. I'd opened a PR yesterday for this, and wanted to reach out about that: https://github.com/Azure/AgentBaker/pull/3075

Reply:
I can't remove v0.5.2 as this is used for Kubernetes version 1.22. i.e. old customer will still be using it.
I can see if I can remove v0.6.1, as after this change Kubernetes version 1.24 will use v0.6.3.

Reply:
Caching components in the VHD is an optimization, for faster pod startup times. Since k8s 1.22 is currently out of support on AKS, caching v0.5.2 may not be necessary.